roles/serviceassurance/tasks/component_qdr.yml

```diff
@@ -33,14 +33,13 @@
             saslMechanisms: ANONYMOUS
             sslProfile: openstack
         interRouterListeners:
-          - authenticatePeer: true
-            expose: true
+          - expose: true
             port: 55671
-            saslMechanisms: EXTERNAL
-            sslProfile: inter-router
+            saslMechanisms: ANONYMOUS
```

Comment:
At first glance I'd expect this to break the inter-router connections that automatically get generated when you ask the operator for >1 QDR. Did you test that?

Reply:
I didn't test that actually. I'm thinking I might need to actually create a separate inbound connection because I hadn't thought about the multi-QDR scenario. I will review today and do some additional testing.